src/sentry/demo/tasks.py

```diff
@@ -0,0 +1,40 @@
+from datetime import timedelta
+from django.conf import settings
+from django.db.models import F
+from django.utils import timezone
+
+from sentry.models import Organization, OrganizationStatus, User
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.deletion import delete_organization
+
+
+MAX_RETRIES = 5
+
+
+@instrumented_task(
+    name="sentry.demo.tasks.delete_users_orgs",
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES,
```

Comment:
Not really sure if I need to worry about retries here

Reply:
Probably not necessary since it's a regularly scheduled task.